src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/CertificationCard.vue

```diff
@@ -4,19 +4,23 @@
       <v-row>
         <v-col :cols="mdAndUp && isCertificateActive ? 8 : 12">
           <div class="d-flex flex-column ga-5">
-            <h1>Early Childhood Educator - {{ titleArray.join(" ") }}</h1>
+            <h1><certification-title :certification="certification" /></h1>
```

Comment:
Nitpik, the component name should it be capitalized and match the component? 

ex. 
<CertificationTitle> 

it would help us distinguish between our custom components versus ones that come with vuetify <v-card> or html <div>